srv/components/misc_pkgs/ipmi/bmc_watchdog/start.sls

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Start and enable bmc-watchdog:
+  service.running:
+    - name: bmc-watchdog
+    - enable: True
+
+Verify bmc-watchdog:
```

Comment:
This should go into a separate `sanity_check.sls`.

Reply:
That's how I'd added it in previous commit, but I have placed a require condition in start file. only when it is enabled, we can check the command. So, added it to start.sls directly.

Reply:
No it has to be in sanity_check.sls.
you can use `include` to set the dependency on start.sls from sanity.sls